tenets/codelingo/go/ignore-goroutine-return-values/codelingo.yaml

```diff
@@ -0,0 +1,19 @@
+tenets:
+  - name: dont-ignore-gofunc-returns
+    actions:
+      codelingo/review:
+        comment: Do not ignore values returned by goroutines.
+    query: |
+      import codelingo/ast/go
+
+      @review comment
+      go.go_stmt(depth = any):
+        go.call_expr:
+          any_of:
+            go.func_lit:
+              go.func_type:
+                go.field_list
+                go.field_list
+            edge("calls"):
+              go.func_decl:
+                go.return_stmt(depth = any)
```

Comment:
I'm guessing this means that if you have a `return` in a `func_decl` we assume that it returns a value? You can return without returning a value. [pg link for](https://play.golang.org/p/gf5aXD7zjY5):

```go
package main

import (
	"fmt"
)

func main() {
	fmt.Println("Hello, playground")
	if true {
		return	
	}
	fmt.Println("Hello, playground")
}
```

Probably safer to use that same pattern as above:

```yaml
go.func_decl:
  go.func_type:
    go.field_list
    go.field_list
```